src/push-state-tree.js

```diff
@@ -56,6 +51,59 @@ const CHANGE = 'change';
 const MATCH = 'match';
 const OLD_MATCH = 'oldMatch';
 
+// Internal history keep tracking of all changes in the location history, it can be reset any
+let internalHistory;
```

Comment:
Will be removed when doing this implementation: https://github.com/gartz/pushStateTree/issues/47